.gitignore

```diff
@@ -2,7 +2,8 @@
 BDPointSDK.framework
 Pods
 *.xcworkspacedata
-
+*.xcworkspace
+xcuserdata/
 
 # Created by https://www.gitignore.io/api/osx,xcode,appcode,cocoapods
```

Comment:
we probably better regenerate `.gitignore` on https://www.gitignore.io
It should add the above changes

Reply:
Regenerated gitignore from https://www.gitignore.io